assets/js/klarna-payments.js

```diff
@@ -159,7 +160,7 @@ jQuery( function ( $ ) {
 			}
 
 			var klarna_payments_container_selector_id = "#" + klarna_payments.getSelectorContainerID()
-			console.log( klarna_payments_container_selector_id )
+			this.debug && console.debug( klarna_payments_container_selector_id )
```

Comment:
Instead of having the check for if debug is enabled or not on each row that we log, id suggest making a function that basically wraps console.log/debug/error with the check instead so we can just call the internal function instead.